src/Filter/FilterComposite.php

```diff
@@ -15,33 +15,31 @@
 class FilterComposite implements FilterInterface
 {
     /**
-     * @var ArrayObject
+     * Constant to add with "or" condition
      */
-    protected $orFilter;
+    public const CONDITION_OR = 1;
 
     /**
-     * @var ArrayObject
+     * Constant to add with "and" condition
      */
-    protected $andFilter;
+    public const CONDITION_AND = 2;
 
     /**
-     * Constant to add with "or" condition
+     * @var ArrayObject
      */
-    const CONDITION_OR = 1;
+    protected $andFilter;
 
     /**
-     * Constant to add with "and" condition
+     * @var ArrayObject
      */
-    const CONDITION_AND = 2;
+    protected $orFilter;
 
     /**
      * We can pass a list of OR/AND filters through construct
      *
```

Comment:
This is a case where the docblock should stay: iterable and array types MUST be documented